WordPress/src/main/java/org/wordpress/android/ui/domains/management/DomainManagementViewModel.kt

```diff
@@ -84,14 +84,14 @@ class DomainManagementViewModel @Inject constructor(
     }
 
     fun onDomainTapped(domain: String, detailUrl: String) {
-        analyticsTracker.track(Stat.DOMAIN_MANAGEMENT_MY_DOMAINS_SCREEN_DOMAIN_TAPPED)
+        analyticsTracker.track(Stat.DOMAIN_MANAGEMENT_DOMAIN_DETAILS_WEB_VIEW_SHOWN)
```

Comment:
Was this meant to be a "tapped" event, rather than a "shown" event (since it hasn't been shown yet at this point)?

Reply:
Good point @mkevins 👍 
I tried to align with the spec and iOS on this but you are right the as is in reality it tracks the tap event and not the details webview open event. I moved the event in the `DomainManagementDetailsActivity` with 9bea904820414fe7fcd2a063d914cef81943e22c